install/tools/man/ipa-migrate.1

```diff
@@ -0,0 +1,112 @@
+.\"
+.\" Copyright (C) 2023  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-migrate" "1" "Fed 1 2024" "IPA" "IPA Manual Pages"
+.SH "NAME"
+ipa\-migrate \- Migrate an IPA server from one machine to another
+.SH "SYNOPSIS"
+ipa\-migrate
+.SH "DESCRIPTION"
+
+Use the \fIipa-migrate\fR command to migrate one
+IPA server to an existing local IPA server installation.
+
+Migrate IPA schema, configuration, and database to a local IPA server.  This
+migration can be done online, where the tool will query the remote server. Or,
+offline where LDIF files can be provided.  You can mix and match online and
+offline. So for example you could migrate the schema and configuration online,
+and then use an exported LDIF file for the database migration portion (this
+might be more useful for very large databases as you don't need to worry about
+network interruptions)
```

Comment:
I think a block about how the server is left in migration-mode so that the migrated users can more easily reset their passwords either by authenticating via SSSD or using the web-based migration page. If we want to get really technical they need to do an LDAP bind using their password to generate the Kerberos keys.